draft-ietf-scitt-architecture.md

```diff
@@ -218,7 +218,9 @@ Equivocation:
 
 Feed:
 
-: see Subject
+: A collection of Signed Statements, observed by the Transparency Service, related to the `sub` field of the 'CWT_Claims' property of the protected header of Signed Statements.
```

Comment:
```suggestion
: An unordered collection of arbitrary data (artifacts, signed statements, receipts, or other data) related to the text identifiers, observed by the transparency service in the `sub` field of the CWT Claim property of the protected header of Signed Statements.
```

Reply:
Unordered: 
This isn't necessarily so. A transparency service may order them. They may even order by different parameters, which is implementation specific. Stating they're not ordered is an over-statement.
This looks like a lot of extra text, to demonstrate we really need to solve what gets persisted, and what gets returned.

Reply:
And, this same text has to be fixed whether it's on the feed or the subject property.